### PR TITLE
KEYCLOAK-16679 Add algorithm settings for client assertion signature in OIDC identity broker

### PR DESCRIPTION
**Pull request by i7a7467**
https://issues.redhat.com/browse/KEYCLOAK-16679

### CODE REVIEW

**Comment by hmlnarik** [comment]
Code PR: keycloak/keycloak#7691

**Comment by mposolda** [comment]
This PR looks good to me and from my point of view, it can be merged once the codebase PR is merged.

@andymunro Could you please also review this PR?

File: server_admin/topics/identity-broker/oidc.adoc [hunk 1]
```diff
@@ -45,6 +45,10 @@ You must define the OpenID Connect configuration options as well.  They basicall
 |Client Secret
 |This realm will need a client secret to use when using the Authorization Code Flow. The value of this field can refer a value from an external <<_vault-administration,vault>>.
 
+|Client Assertion Signature Algorithm
+|Signature algorithm to create JWT assertion as client authentication.
+In the case of JWT signed with private key or Client secret as jwt, it is required. If no algorithm is specified, the following algorithm is adapted. `RS256` is adapted the in the case of JWT signed with private key.  `HS256` is adapted the in the case of Client secret as jwt.
```
Comment (andymunro) [inline server_admin/topics/identity-broker/oidc.adoc]: remove "the" before both occurrences of "in"

**Review by andymunro — CHANGES_REQUESTED** [review]
Just one line to change. Thanks.

File: server_admin/topics/identity-broker/oidc.adoc (hunk 1 above)
Comment (i7a7467) [reply server_admin/topics/identity-broker/oidc.adoc]: @mposolda @andymunro 
Thanks. I fixed your review point.

**Review by mposolda — APPROVED** [review]
@ryandawsonuk @andymunro Thanks for the PR and for the review!

**Pull request merged**